### ISSUE TITLE
Fix this source mtlnation.com

### ISSUE THREAD

**Issue by read-novel** [dipu-bd/lightnovel-crawler#2295]
## Let us know

**Novel URL**:     > 
**App Location**:     EXE app in laptop 💻 

**App Version**:   x.y.z  <!- lightnovel crawler  v.3.5.0->

## Describe this issue
When I use bot to download the chapter from mtlnation.com below 100 chapter was fine but above the 100 chapter were only half. It say that you need to login to continue, so I use crawler app in laptop to get complete chapter but whenever I fill the username and password it say error. I have uploaded screenshot. I hope you can fix 😢
![IMG_20240305_200324](https://github.com/dipu-bd/lightnovel-crawler/assets/107703937/b41b93c0-8025-4ebe-bdf6-f7d123ad9f91)

 unable to get complete chapter above 100 from mtlnation.com website

**Comment by read-novel** [comment]
This is link to novel which I want to download 
https://mtlnation.com/novel/after-ten-reincarnations-all-the-heroines-have-becom

**Comment by CyberCrisis5** [comment]
Just to say that if you do not login the chapters after a certain number get cut almost in half.

**Comment by CryZFix** [comment]
> username and password

To log in, you need to use a JWT token similar to the instructions for Wuxia >> [Click](https://github.com/dipu-bd/lightnovel-crawler/discussions/1360#discussion-4074467) <<
After logging in, you need to find a request with the name "me" and copy the token from it, then pass it to the crawler

**Comment by CyberCrisis5** [comment]
I tried using the the command lncrawl --login JWT eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MTIzNzI2NzcsImp0aSI6IjEwNTMyMSJ9.ViaAsF
1r4AdJuRxzKfxat1As24keN0PII-MR9aM_9oE

but the chapters still get cut off it just does not ask for login

**Comment by CryZFix** [comment]
> I tried using the the command lncrawl --login JWT eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MTIzNzI2NzcsImp0aSI6IjEwNTMyMSJ9.ViaAsF 1r4AdJuRxzKfxat1As24keN0PII-MR9aM_9oE
> 
> but the chapters still get cut off it just does not ask for login

Please check if you have any errors in your token. You can also check if you have logged in by adding the `-lll` parameter for logging. After specifying the link, you should see the line `Login with ['JWT', 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9...']` A couple of lines down you will see the line with your login to the site: `Logged in as "your_username"`.

**Comment by CyberCrisis5** [comment]
This is the first time I used this program so I am not sure what you mean by -111 so am sure I am using it wrong because I never get the right response. I log out and back in to get a new token and here is the new line I used lncrawl --login JWT eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MTI0MDg2MDQsImp0aSI6IjEwNTMyMSJ9.Y1UOGHj_dlU5nSvf2ddXfaB3fqPRiGQ4vqKC18bsTe4 -111 plus thanks for helping me.

**Comment by read-novel** [comment]
> Just to say that if you do not login the chapters after a certain number get cut almost in half and it ask you to login at the bottom to see the rest.

Yes, after some conclusions i think above chapter 100 are not completely downloaded from mtlnation.com. even other file are same if you downloading them using lightnovel crawler bot. It say: please, login to continue reading 
![IMG_20240307_210505](https://github.com/dipu-bd/lightnovel-crawler/assets/107703937/c229d80d-1599-42e9-ad0d-321ff54ddc6e)
As you can see in the screenshot, please help

**Comment by dipu-bd** [comment]
Closing as a duplicate of #1961, which is the oldest open report for `mtlnation.com` and is where this is tracked.

Nothing is lost by this — subscribe to #1961 for updates. If what you hit is actually a different fault on the same site rather than the same one, say so and this will be reopened.
